lib/sisimai/bite/email/google.rb

```diff
@@ -0,0 +1,320 @@
+        '12' => { 'command' => 'AUTH', 'reason' => 'relayingdenied' },
+
+        # Technical details of permanent failure:
+        # Google tried to deliver your message, but it was rejected by the recipient domain.
+        # We recommend contacting the other email provider for further information about the
+        # cause of this error. The error that the other server returned was:
+        # 550 550 5.7.1 <****@gmail.com>... Access denied (state 13).
+        '13' => { 'command' => 'EHLO', 'reason' => 'blocked' },
+
+        # Technical details of permanent failure:
+        # Google tried to deliver your message, but it was rejected by the recipient domain.
+        # We recommend contacting the other email provider for further information about the
+        # cause of this error. The error that the other server returned was:
+        # 550 550 5.1.1 <******@*********.**>... User Unknown (state 14).
+        # 550 550 5.2.2 <*****@****.**>... Mailbox Full (state 14).
+        #
+        '14' => { 'command' => 'RCPT', 'reason' => 'userunknown' },
+
+        # http://www.google.cz/support/forum/p/gmail/thread?tid=7090cbfd111a24f9&hl=en
+        # Technical details of permanent failure:
+        # Google tried to deliver your message, but it was rejected by the recipient domain.
+        # We recommend contacting the other email provider for further information about the
+        # cause of this error. The error that the other server returned was:
+        # 550 550 5.7.1 SPF unauthorized mail is prohibited. (state 15).
+        # 554 554 Error: no valid recipients (state 15).
+        '15' => { 'command' => 'DATA', 'reason' => 'filtered' },
+
+        # http://www.google.com/support/forum/p/Google%20Apps/thread?tid=0aac163bc9c65d8e&hl=en
+        # Technical details of permanent failure:
+        # Google tried to deliver your message, but it was rejected by the recipient domain.
+        # We recommend contacting the other email provider for further information about the
+        # cause of this error. The error that the other server returned was:
+        # 550 550 <****@***.**> No such user here (state 17).
+        # 550 550 #5.1.0 Address rejected ***@***.*** (state 17).
+        '17' => { 'command' => 'DATA', 'reason' => 'filtered' },
+
+        # Technical details of permanent failure:
+        # Google tried to deliver your message, but it was rejected by the recipient domain.
+        # We recommend contacting the other email provider for further information about the
+        # cause of this error. The error that the other server returned was:
+        # 550 550 Unknown user *****@***.**.*** (state 18).
+        '18' => { 'command' => 'DATA', 'reason' => 'filtered' },
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Google Gmail: https://mail.google.com'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
```

Comment:
__[rubocop]__
* Layout/EmptyLineBetweenDefs: Use empty lines between method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods)
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->